_health-care/_js/_components/gender-input.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import _ from 'lodash';
+
+class GenderInput extends React.Component {
+  constructor() {
+    super();
+    this.state = { hasError: false };
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  componentWillMount() {
+    this.id = _.uniqueId();
+  }
+
+  handleChange() {
+    const gender = this.refs.gender.value;
+
+    if (this.validate(gender)) {
+      this.setState({ hasError: false });
+    } else {
+      this.setState({ hasError: true });
+    }
+
+    this.props.onUserInput(gender);
+  }
+
+  validate(field) {
+    if (field === '') {
```

Comment:
This raises a good question. I don't have a sense right now of what fields are required or not. I don't know if we can assume that only fields with the "Required" span text to it are required. It seems to me that gender should be a required field. But if it's not required, I don't think we should have to run validations at all.